pkg/descheduler/evictions/evictions_test.go

```diff
@@ -117,51 +144,152 @@ func TestPodTypes(t *testing.T) {
 	}
 }
 
-func TestNewPodEvictor(t *testing.T) {
+func TestPodEvictor(t *testing.T) {
 	pod1 := test.BuildTestPod("pod", 400, 0, "node", nil)
-
-	fakeClient := fake.NewSimpleClientset(pod1)
-
-	eventRecorder := &events.FakeRecorder{}
-
-	podEvictor := NewPodEvictor(
-		fakeClient,
-		eventRecorder,
-		NewOptions().WithMaxPodsToEvictPerNode(utilptr.To[uint](1)),
-	)
-
-	stubNode := &v1.Node{ObjectMeta: metav1.ObjectMeta{Name: "node"}}
-
-	// 0 evictions expected
-	if evictions := podEvictor.NodeEvicted(stubNode); evictions != 0 {
-		t.Errorf("Expected 0 node evictions, got %q instead", evictions)
+	type podEvictorTest struct {
+		description                string
+		pod                        *v1.Pod
+		dryRun                     bool
+		maxPodsToEvictTotal        *uint
+		maxPodsToEvictPerNode      *uint
+		maxPodsToEvictPerNamespace *uint
+		expectedNodeEvictions      uint
+		expectedTotalEvictions     uint
+		expectedError              error
```

Comment:
s/expectedError/err as expected is used for comparison with actual. s/expectedError/actualErr will work as well.

Reply:
replace err -> actualErr